app/v1/validation/fullReturn/AdjustedGroupInterestValidator.scala

```diff
@@ -63,6 +63,8 @@ trait AdjustedGroupInterestValidator extends BaseValidation {
         adjustedGroupInterestModel.groupRatio.validNec
       case (_, groupEBITDA, groupRatio) if groupEBITDA <= 0 && groupRatio != 100 =>
         NegativeOrZeroGroupEBITDAError(groupRatio).invalidNec
+      case (qngie, _, groupRatio) if qngie <= 0 && groupRatio != 100 =>
```

Comment:
So the qngie can't be less than 0 regardless of the value of the groupRatio. We probably want this as a separate piece of validation rather than in this match statement so that it will be output alongside any errors found in this method. Similar to how we have `validateGroupEBITDA` separate.